template/templates/Containerfile.j2

```diff
@@ -1,4 +1,4 @@
-{%- let os_version = recipe.get_os_version() %}
+{%- let os_version = self.os_version %}
```

Comment:
I don't think this is valid syntax. You would need to add the `os_version` to add this to `ContainerFileTemplate` and then you can remove this line entirely.

Reply:
We can just delete this. I'm an idiot, you're already doing that.
```suggestion
```